code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -196,6 +196,7 @@
 		/obj/item/cultivator,
 		/obj/item/hatchet,
 		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/glass/jug,
```

Comment:
Also an undocumented CL. Best to be as detailed as you can. Or put them in one add CL